Automattic/wp-calypso
# My Plan: Add support for plan icons for real-time Backup and Security

Resolves `1200412004370260-as-1201306365600957`, `p1HpG7-dpL-p2#comment-50025`.

#### Changes proposed in this Pull Request

* Add slugs for new real-time Backup and Security plans to the product-icon map, so that the correct icon shows up on the My Plan page for these new offerings.

#### Testing instructions

**NOTE:** This PR should be tested with both 10GB and 1TB version of Backup and Security, in both monthly and yearly billing terms. **Only visible on screen layouts >660px wide.**

1. In Calypso Blue, visit the My Plan page under **Upgrades > Plans**.
2. Verify the correct icon shows up alongside the plan name (see reference screenshots).

#### Reference screenshots

<img width="1053" alt="image" src="https://user-images.githubusercontent.com/670067/139901746-75bb9f1c-a606-414c-b21c-f025e069f554.png">

<img width="1051" alt="image" src="https://user-images.githubusercontent.com/670067/139904092-32ef90a2-59af-4ff7-8eb0-38052408db5f.png">
